server/category.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/labstack/echo/v4"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func GetCat(c echo.Context) error {
```

Comment:
These functions should take in a collection (remember these are your own functions that you can do as you please with).